### PR TITLE
Monkeys now use height offset (and monkey tail works)

### LINKED ISSUE #64466: Monkeys who lose their tail organ still appear to have a tail
Reporting client version: 514.1575

## Round ID:
[177356](https://scrubby.melonmesa.com/round/177356)

## Testmerges:
- [Refactors pay stands + custom vendors](https://www.github.com/tgstation/tgstation/pull/63889)

## Reproduction:
image

### LINKED ISSUE #75990: ERROR monkey
obraz

## Round ID:
[208030](https://scrubby.melonmesa.com/round/208030)

## Testmerges:
[#75946: 'Pink Security' by Youtubeboy139 at commit e0cbe67e0f](https://www.github.com/tgstation/tgstation/pull/75946)

## Reproduction:
Neither me, nor an admin have any idea how this happened. I found it rapidly switching places with other non-sentient monkeys inside a depowered genetics pen.
Mind magnifying it, made the model reappear, but the ERROR remained over it.

Mob ID (provided by Justice): @mob_3584

### LINKED ISSUE #76353: Monkey bodypart overlays runtime to hell and back
## Round ID: [208572](https://scrubby.melonmesa.com/round/208572) [209087](https://scrubby.melonmesa.com/round/209087)
most recent rounds, visit [this moth fans tracker](https://runtimes.moth.fans/runtime/Cannot%20read%20null.icon_state_______%2Fdatum%2Fbodypart_overlay%2Fmutant%2Ftail%2Fget_base_icon_state) for more

## Runtimes
Seems like the runtimes are spread across 4 variants

### Example runtime №1: `Cannot read null.icon_state`
<details><summary>Click</summary>

```
[2023-06-26 00:19:32.894] RUNTIME: runtime error: Cannot read null.icon_state
 - proc name: get base icon state (/datum/bodypart_overlay/mutant/tail/get_base_icon_state)
 -   source file: tails.dm,75
 -   usr: Clara White (/mob/living/carbon/human)
 -   src: /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey)
 -   usr.loc: the floor (158,102,4) (/turf/open/floor/iron/white)
 -   call stack:
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get base icon state()
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): generate icon cache()
 - the monkey chest (/obj/item/bodypart/chest/monkey): generate icon key()
 - Clara White (/mob/living/carbon/human): update body parts(null)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - Monkey (/datum/species/monkey): regenerate organs(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 1, null, 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): finish monkeyize()
 - /datum/callback (/datum/callback): Invoke()
 - world: push usr(Clara White (/mob/living/carbon/human), /datum/callback (/datum/callback))
 - /datum/callback (/datum/callback): InvokeAsync()
 - Timer (/datum/controller/subsystem/timer): fire(0)
 - Timer (/datum/controller/subsystem/timer): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
 - 
```

</details> 

### Example runtime №2: `Trying to call get_image()...`
<details><summary>Click</summary>

```
[2023-06-26 00:19:32.896] RUNTIME: runtime error: Trying to call get_image() on /datum/bodypart_overlay/mutant/tail/monkey while it didn't have a sprite_datum. This shouldn't happen, report it as soon as possible.
 - proc name: get image (/datum/bodypart_overlay/mutant/get_image)
 -   source file: mutant_bodypart_overlay.dm,48
 -   usr: Clara White (/mob/living/carbon/human)
 -   src: /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey)
 -   usr.loc: the floor (158,102,4) (/turf/open/floor/iron/white)
 -   call stack:
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get image(-6, the monkey chest (/obj/item/bodypart/chest/monkey))
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get overlay(-6, the monkey chest (/obj/item/bodypart/chest/monkey))
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get overlay(1, the monkey chest (/obj/item/bodypart/chest/monkey))
 - the monkey chest (/obj/item/bodypart/chest/monkey): get limb icon(null)
 - Clara White (/mob/living/carbon/human): update body parts(null)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - Monkey (/datum/species/monkey): regenerate organs(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 1, null, 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): finish monkeyize()
 - /datum/callback (/datum/callback): Invoke()
 - world: push usr(Clara White (/mob/living/carbon/human), /datum/callback (/datum/callback))
 - /datum/callback (/datum/callback): InvokeAsync()
 - Timer (/datum/controller/subsystem/timer): fire(0)
 - Timer (/datum/controller/subsystem/timer): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
```
</details> 

### Example runtime №3: `Cannot read null.color_src`
<details><summary>Click</summary>

```
[2023-06-26 00:19:32.898] RUNTIME: runtime error: Cannot read null.color_src
 - proc name: color image (/datum/bodypart_overlay/mutant/color_image)
 -   source file: mutant_bodypart_overlay.dm,68
 -   usr: Clara White (/mob/living/carbon/human)
 -   src: /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey)
 -   usr.loc: the floor (158,102,4) (/turf/open/floor/iron/white)
 -   call stack:
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): color image(null, -6, the monkey chest (/obj/item/bodypart/chest/monkey))
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get overlay(-6, the monkey chest (/obj/item/bodypart/chest/monkey))
 - /datum/bodypart_overlay/mutant... (/datum/bodypart_overlay/mutant/tail/monkey): get overlay(1, the monkey chest (/obj/item/bodypart/chest/monkey))
 - the monkey chest (/obj/item/bodypart/chest/monkey): get limb icon(null)
 - Clara White (/mob/living/carbon/human): update body parts(null)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - Monkey (/datum/species/monkey): regenerate organs(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 1, null, 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): finish monkeyize()
 - /datum/callback (/datum/callback): Invoke()
 - world: push usr(Clara White (/mob/living/carbon/human), /datum/callback (/datum/callback))
 - /datum/callback (/datum/callback): InvokeAsync()
 - Timer (/datum/controller/subsystem/timer): fire(0)
 - Timer (/datum/controller/subsystem/timer): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
```

</details> 

### Example runtime №4: `Cannot read null.pixel_y`
<details><summary>Click</summary>

```
[2023-06-26 00:19:32.903] RUNTIME: runtime error: Cannot read null.pixel_y
 - proc name: update body parts (/mob/living/carbon/proc/update_body_parts)
 -   source file: carbon_update_icons.dm,527
 -   usr: (src)
 -   src: Clara White (/mob/living/carbon/human)
 -   src.loc: the floor (158,102,4) (/turf/open/floor/iron/white)
 -   call stack:
 - Clara White (/mob/living/carbon/human): update body parts(null)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - the tail (/obj/item/organ/external/tail/monkey): Insert(Clara White (/mob/living/carbon/human), 1, 0)
 - Monkey (/datum/species/monkey): regenerate organs(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 1, null, 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Monkey (/datum/species/monkey): on species gain(Clara White (/mob/living/carbon/human), Human (/datum/species/human), 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): set species(/datum/species/monkey (/datum/species/monkey), 1, 0)
 - Clara White (/mob/living/carbon/human): finish monkeyize()
 - /datum/callback (/datum/callback): Invoke()
 - world: push usr(Clara White (/mob/living/carbon/human), /datum/callback (/datum/callback))
 - /datum/callback (/datum/callback): InvokeAsync()
 - Timer (/datum/controller/subsystem/timer): fire(0)
 - Timer (/datum/controller/subsystem/timer): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
```

</details>

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This PR adds the ability for monkeys to wear any jumpsuit in the game, and adds support for them to wear things like coats, gloves, and shoes (though this cannot be obtained in-game and is solely achieved through admins, which I also improved a bit upon by adding a defined bitfield for no equip flags).

This reverts a lot of changes from https://github.com/tgstation/tgstation/pull/73325 - We no longer check height from limbs and such to bring sprites down, instead monkeys now work more similarly to humans, so the entire PR was made irrelevant, and I didn't really want to leave around dead code for the sake of having a human with longer legs.

I've now also added support for Dwarfism, which makes monkeys look even smaller. Very minor change but at least now the mutation doesn't feel like it does literally nothing to monkeys (since they can already walk over tables).

Here's a few examples of how it can appear in game (purely for demonstration, as it is currently intentionally made impossible to obtain in-game, though if someone wants to change that post-this PR now that support is added, feel free):

Human with all Monkey limbs
image
image
image

Monkey with all Human limbs
image

Augmented Monkeys
image
image

Pretty cool, unfortunately just not my field of expertise.

Anyway, here's a video demonstration of me abusing monkey clothing.

https://github.com/tgstation/tgstation/assets/53777086/70f47a7e-0c86-470a-8b8e-7b5d85e54e42

### Tails

Tails have been broken for a while now, the only reason you see them in-game is because they are baked into the monkey sprites. This fixes that, which means humans can now get monkey tails implanted into them (hell yeah) and monkeys can have their tails removed (also hell yeah)

https://github.com/tgstation/tgstation/assets/53777086/2d857a60-e8de-4eea-a0e2-437ffd8bd68b

## Why It's Good For The Game

Monkeys are unironically my favorite species and I wanted to expand on clothings monkeys can wear since https://github.com/tgstation/tgstation/pull/68489 - but then Melbert dropped https://github.com/tgstation/tgstation/pull/72344 and while it is not perfect (as you can see in the videos), I tried making it look as best as I personally could, and hopefully in the future this can be improved upon.
It's annoyingly limiting to players that they can only wear 2 jumpsuits in the whole game, plus the snowflake code they have was a little meh, this hopefully fixes those issues and makes future improvement and general monkeying even better, both for players and admins.

No more should monkeys be considered monkey that happens to be a crewmember, but a crewmember that happens to be a monkey.

Base Monkey
image
image
image
image

With Geneticist jumpsuit
image
image

Pun Pun on roundstart
image

Closes https://github.com/tgstation/tgstation/issues/64466
Closes https://github.com/tgstation/tgstation/issues/76353
Closes https://github.com/tgstation/tgstation/issues/75990 (fixing the hands)
Fixes half of https://github.com/tgstation/tgstation/issues/76938

## Changelog

:cl: JohnFulpWillard, Tattax
add: Monkeys can now wear all jumpsuits.
add: Bioscrambler anomalies can now give monkey legs.
fix: Monkeys can have their tails removed, and tails are no longer invisible when implanted into another species.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/_species.dm [hunk 1]
```diff
@@ -432,7 +433,8 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			wearer.dropItemToGround(equipped_item, force = TRUE)
 
 /datum/species/proc/update_no_equip_flags(mob/living/carbon/wearer, new_flags)
-	no_equip_flags = new_flags
+	if(new_flags)
+		no_equip_flags = new_flags
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/_species.dm:437]: This means you can't update a species equip flags to `NONE`?

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 2]
```diff
@@ -271,8 +271,10 @@
 /mob/living/carbon/human/proc/set_mob_height(new_height)
 	if(mob_height == new_height)
 		return FALSE
-	if(new_height == HUMAN_HEIGHT_DWARF)
-		CRASH("Don't set height to dwarf height directly, use dwarf trait")
+	if(new_height == HUMAN_HEIGHT_DWARF || new_height == MONKEY_HEIGHT_DWARF)
+		CRASH("Don't set height to dwarf height directly, use dwarf trait instead.")
+	if(new_height == new_height == MONKEY_HEIGHT_MEDIUM)
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/human_helpers.dm:276]:
```suggestion
	if(new_height == MONKEY_HEIGHT_MEDIUM)
```

File: code/modules/mob/living/carbon/human/_species.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/mob/living/carbon/human/_species.dm:437]: You shouldn't be anyways, I intentionally left out belt/pockets/ID since it should be tied to the jumpsuit instead.

Comment (JohnFulpWillard) [reply code/modules/mob/living/carbon/human/_species.dm:437]: ah fuck you're right i was thinking backwards

File: code/modules/mob/living/carbon/human/species_types/abominations.dm [hunk 3]
```diff
@@ -1,18 +1,4 @@
 /// These won't appear normally in games, they are meant to for debuging the adjustment of limbs based on the height of a humans bodyparts.
-/datum/species/human/tallboy
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/species_types/abominations.dm:2]: Should delete all of these species

Comment (JohnFulpWillard) [reply code/modules/mob/living/carbon/human/species_types/abominations.dm:2]: removed the other but left the one that's on birdshot
image

Comment (MrMelbert) [reply code/modules/mob/living/carbon/human/species_types/abominations.dm:2]: IMO just delete the birdshot thing and replace it with a monkey, or if you want to go through the effort, add a mapping helper which replaces limbs of the mob it's placed on

These are just not good species

**Comment by MrMelbert** [comment]
Also of al the pictures you included in the PR you didn't actually include any pictures of what monkeys just, look like, in general

**Comment by IndieanaJones** [comment]
The base monkey in the PR body looks... Bad? Its legs aren't properly attached and the tail sprite is getting clipped.

**Comment by moocowswag** [comment]
monkey with lizard limbs, we can finally get kobolds!!!

**Comment by JohnFulpWillard** [comment]
I resized monkey limbs so they can theoretically fit on humans better (though still not supported properly), which fixes the tail too. I'll upload screenshots in a bit I accidentally closed my localhost early and have to rebuild.

**Comment by 13spacemen** [comment]
@JohnFulpWillard 
good PR and i realize that its impossible for auto-fitting sprites to look perfect

but i really think that IF species sprites exist, it should default to those [icon_exists()], else do the height map displacement

that way Pun Pun, who has proper monkey clothes still looks the same

i know the bodytype checking code is a bit 'snowflake' but we can refactor it to be neat and tidy later, baycode servers only have one check for that sort of thing

but i really think you should give priority to species sprites if they exist

**Review by Wallemations — CHANGES_REQUESTED** [review]
1) why are the monkey's legs dislocated
2) why are they wearing wings when they put on jumpsuits, also where are their arms

**Comment by JohnFulpWillard** [comment]
> @JohnFulpWillard good PR and i realize that its impossible for auto-fitting sprites to look perfect
> 
> but i really think that IF species sprites exist, it should default to those [icon_exists()], else do the height map displacement
> 
> that way Pun Pun, who has proper monkey clothes still looks the same
> 
> i know the bodytype checking code is a bit 'snowflake' but we can refactor it to be neat and tidy later, baycode servers only have one check for that sort of thing
> 
> but i really think you should give priority to species sprites if they exist

The problem with it is that it would be incredibly hard and painful to only exempt mob height only if they are wearing this one specific type of jumpsuit, plus it wouldnt take into account dwarfism. I think our best bet is to try to get these jumpsuits looking best as possible but I don't think we can continue to use the old sprites.

**Comment by JohnFulpWillard** [comment]
I fixed hands, turns out they had the same issue as tails of being backed into the arms rather than separate sprites like all other species
image

**Comment by optimumtact** [comment]
what is the client cost of these displacement maps

**Comment by JohnFulpWillard** [comment]
> what is the client cost of these displacement maps

I've been messing around with it and it doesn't seem bothersome, spamming monkeys causes lag because of AIs being laggy, but spamming AI-less ones seem just fine.

**Comment by MrMelbert** [comment]
Stale b gone because I still wanna see this testmerged and testmerges are still funky

**Comment by Time-Green** [comment]
Let me know when you’re satisfied and I’ll review it

File: code/datums/datum.dm [hunk 4]
```diff
@@ -313,6 +313,16 @@
 	filter_data[name] = copied_parameters
 	update_filters()
 
+///A version of add_filter that takes a list of filters to add rather than being individual, to limit calls to update_filters().
+/datum/proc/add_filters(list/filters)
+	LAZYINITLIST(filter_data)
+	for(var/list/individual_filter in filters)
```
Comment (Ghommie) [inline code/datums/datum.dm:319]:
```suggestion
	for(var/list/individual_filter as anything in filters)
```

File: code/modules/admin/player_panel.dm [hunk 5]
```diff
@@ -231,10 +231,10 @@
 			if(isliving(M))
 
 				if(iscarbon(M)) //Carbon stuff
-					if(ismonkey(M))
-						M_job = "Monkey"
-					else if(ishuman(M))
+					if(ishuman(M) && M.job)
 						M_job = M.job
+					else if(ismonkey(M))
+						M_job = "Monkey"
```
Comment (Ghommie) [inline code/modules/admin/player_panel.dm:237]: Monkeys are now a species datum, while the atom path is `carbon/human`, so this `else` conditional will never pass.

**Comment by LemonInTheDark** [comment]
@MrMelbert MSO has a guide posted in keybus it's very trivial to tm, unless the other servers are filled up with other stuff you should throw this up somewhere

File: code/modules/admin/player_panel.dm (hunk 5 above)
Comment (LemonInTheDark) [reply code/modules/admin/player_panel.dm:237]: brb fucking dying

File: code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm [hunk 6]
```diff
@@ -84,21 +84,6 @@
 	. = ..()
 	disconnect_from_area(area_to_unregister)
 
-///adds a gas or list of gases to our filter_types. used so that the scrubber can check if its supposed to be processing after each change
-/obj/machinery/atmospherics/components/unary/vent_scrubber/proc/add_filters(filter_or_filters)
```
Comment (Time-Green) [inline code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm:88]: What does this have to do with the PR?

**Review by Time-Green — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm (hunk 6 above)
Comment (JohnFulpWillard) [reply code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm:88]: I added a proc called ``add_filters`` and this was using the same name and was completely unused so i just deleted it

File: code/datums/datum.dm [hunk 7]
```diff
@@ -313,6 +313,16 @@
 	filter_data[name] = copied_parameters
 	update_filters()
 
+///A version of add_filter that takes a list of filters to add rather than being individual, to limit calls to update_filters().
+/datum/proc/add_filters(list/filters)
```
Comment (MrMelbert) [inline code/datums/datum.dm:317]:
```suggestion
/datum/proc/add_filters(list/list/filters)
```

File: code/modules/mob/living/carbon/human/_species.dm [hunk 8]
```diff
@@ -624,8 +624,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			var/obj/item/organ/internal/eyes/eye_organ = species_human.get_organ_slot(ORGAN_SLOT_EYES)
 			if(eye_organ)
 				eye_organ.refresh(call_update = FALSE)
-				for(var/mutable_appearance/eye_overlay in eye_organ.generate_body_overlay(species_human))
-					eye_overlay.pixel_y += height_offset
+				for(var/eye_overlay in eye_organ.generate_body_overlay(species_human))
 					standing += eye_overlay
```
Comment (MrMelbert) [inline code/modules/mob/living/carbon/human/_species.dm:628]: This can just be `standing += eye_organ.generate_body_overlay(species_human)` now right?

File: code/__DEFINES/DNA.dm [hunk 9]
```diff
@@ -56,8 +56,9 @@
 #define DNA_MOTH_MARKINGS_BLOCK 13
 #define DNA_MUSHROOM_CAPS_BLOCK 14
 #define DNA_POD_HAIR_BLOCK 15
+#define DNA_MONKEY_TAIL_BLOCK 16
```
Comment (MrMelbert) [inline code/__DEFINES/DNA.dm:59]: I remember we talked about this in the past but I'm still not entirely sold on us having a DNA block for monkey tails (given there's only one monkey tail option that can exist). Is it absolutely necessary for monkey tails to function?

**Review by MrMelbert — COMMENTED** [review]
Few small things but otherwise I see no complaints from testing so w/e

File: code/__DEFINES/DNA.dm (hunk 9 above)
Comment (JohnFulpWillard) [reply code/__DEFINES/DNA.dm:59]: It's not needed and I can remove it, I just thought it would be nice to not let them snowflake it, Felinids up until recently also had their own dna block for just the tail and I didn't want monkeys to be excluded without any good reason for it.

**Comment by Toonpay** [comment]
this is the best change ive ever read in a ss13 server

**Comment by Toonpay** [comment]
> Few small things but otherwise I see no complaints from testing so w/e

:trollface:

File: code/__DEFINES/DNA.dm (hunk 9 above)
Comment (MrMelbert) [reply code/__DEFINES/DNA.dm:59]: I just think DNA blocks are a crap system in general tbh

Comment (Toonpay) [reply code/__DEFINES/DNA.dm:59]: :trollface:

Comment (JohnFulpWillard) [reply code/__DEFINES/DNA.dm:59]: I actually forgot the full reason but I removed it and now tails just aren't working

image

Unfortunately it looks like my past self was smarter than me

Comment (MrMelbert) [reply code/__DEFINES/DNA.dm:59]: And this is why I hate DNA blocks

**Comment by SyncIt21** [comment]
Someone remove the stale label. It's been a week since it was added and am not sure  if it will autoclose as a result now.
